src/components/donations/grid/GridAppbar.tsx

```diff
@@ -16,6 +19,19 @@ const addIconStyles = {
 export default function GridAppbar() {
   const router = useRouter()
   const { t } = useTranslation()
+  const exportToExcel = useMutation({
+    mutationFn: useExportToExcel(),
+    onError: () => AlertStore.show(t('common:alerts.error'), 'error'),
+    onSuccess: (response) => {
+      const url = window.URL.createObjectURL(response.data)
+      const anchor = document.createElement('a')
+      anchor.href = url
+      anchor.download = 'Donations.xls'
+      anchor.dispatchEvent(new MouseEvent('click'))
+      window.URL.revokeObjectURL(url)
```

Comment:
@AndreyGoranov We may extract this so it becomes a utility to download files. It might be useful if we do other exports besides donations.

```tsx
const downloadFile = (filename: string, data: string) => {
  const url = window.URL.createObjectURL(data)
  const anchor = document.createElement('a')
  anchor.href = url
  anchor.download = filename
  anchor.dispatchEvent(new MouseEvent('click'))
  window.URL.revokeObjectURL(url)
}

downloadFile('Donations.xls', response.data)
```

Reply:
Good point. Will extract it